Sources/BCrypt/BCrypt.swift

```diff
@@ -603,9 +603,9 @@ public struct BCryptSalt {
     Creates a new random salt with the specified cost factor. Default cost factor of 10, which is probably
     ~100 ms to hash a password on a modern CPU.
     */
-    public init(costFactor: Int?) throws {
+    public init(workFactor: Int? = nil) throws {
```

Comment:
I left out the default value of nil here because the other initializer is declared with a default parameter as `init(cost: Int = 10)`. Therefore, trying to initialize a `BCryptSalt` with an empty parameter list will actually pick the deprecated method instead of the new initializer.

```swift
let salt = try BCryptSalt()
```

Produces warnings:

```
crypto/Tests/BCryptTests/BCryptTests.swift:72:24: 'init(cost:)' is deprecated: Use `init(workFactor: Int?) throws` instead.
crypto/Tests/BCryptTests/BCryptTests.swift:72:20: No calls to throwing functions occur within 'try' expression
```